main.go

```diff
@@ -0,0 +1,202 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/Clever/marathon-stats/marathon"
+	"github.com/Clever/marathon-stats/mesos"
+	"gopkg.in/clever/kayvee-go.v2"
+)
+
+type m map[string]interface{}
+
+var (
+	mesosMasterPort    string
+	mesosMasterHost    string
+	marathonMasterPort string
+	marathonMasterHost string
+	logMarathonTasks   bool
+	pollInterval       time.Duration
+	version            string
+)
+
+func init() {
+	mesosMasterHost = getEnv("MESOS_HOST")
+	mesosMasterPort = getEnv("MESOS_PORT")
+	marathonMasterHost = getEnv("MARATHON_HOST")
+	marathonMasterPort = getEnv("MARATHON_PORT")
```

Comment:
Would it make sense to panic or print error messages if any of these environment var aren't found?